### PR TITLE
Example: Form handler with Redux and React-bootstrap

### PR DESCRIPTION
**Pull request by estrada9166**
The idea with this is to show how easy can be to create a form handler using redux... it let you access to the state and to the formReducer where are going to be the different forms with the inputs of each one. In this case I used react-bootstap but can be used with anyone...
### e.g
```javascript
<Input controlLabel="Name" title="user" name="name" />
<Input controlLabel="Last name" title="user" name="lastName" />
<Input controlLabel="Email" type="email" title="user" name="email" />
<Input controlLabel="Password" type="password" title="user" name="password" />
```
```json
{
  "formReducer": {
    "user": {
      "name": "",
      "lastName": "",
      "email": "",
      "password": ""
    }
  }
}
```

### CODE REVIEW

File: examples/form-handler/handlers/Input.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  render () {
+    return (
+      <div>
+        <FormGroup controlId='formBasicText'>
+          <ControlLabel>{this.props.controlLabel}</ControlLabel>
+          <FormControl
+            disabled={this.props.disabled}
+            type={this.props.type || 'Text'}
+            placeholder={this.props.controlLabel}
+            onChange={(e) => this.inputChange(this.props.title, this.props.name, e)}
```
Comment (plxel) [inline examples/form-handler/handlers/Input.js]: maybe
 `onChange={this.inputChange}`
and
`inputChange = (e) => this.props.inputChange(this.props.title, this.props.name, e.target.value) `?

Comment (estrada9166) [reply examples/form-handler/handlers/Input.js]: The changes has been made

Comment (plxel) [reply examples/form-handler/handlers/Input.js]: @estrada9166 
if you like destructuring, it also can be smth like this:
```
inputChange = (e) => {
 const { 
  inputChange,
  title,
  name,
 } = this.props;

 inputChange(title, name, e.target.value);
}
```

Comment (estrada9166) [reply examples/form-handler/handlers/Input.js]: Done! Thx

**Review by timneutkens — APPROVED** [review]
Seems alright, thanks!

**Pull request merged**